src/mono/mono/mini/CMakeLists.txt

```diff
@@ -381,6 +381,41 @@ if(NOT DISABLE_SHARED_LIBS)
     add_library(monosgen-shared-dac SHARED "mini-windows-dlldac.c")
     set_target_properties(monosgen-shared-dac PROPERTIES OUTPUT_NAME ${MONO_SHARED_LIB_NAME}-dac)
   endif()
+
+
+  if(TARGET_DARWIN)
+    string(REPLACE ";" "*" mono-components-objects-nowhitespace "${mono-components-objects}")
```

Comment:
Is there a way to disable it/only enable it so its not built during local builds ?

Reply:
Could add a property to the csproj and json to turn on all the framework stuff?

Reply:
@directhex I would add it to this PR if it's easy or in a follow up if it's proving to be more tricky.

Reply:
@steveisok already done locally, just need to know what should happen with the packs subset (i.e. should nupkgs be built with no frameworks if `/p:BuildDarwinFrameworks=true` isn't passed, or should the build intentionally fail?)